src/main/scala/io/iohk/ethereum/crypto/ECDSASignature.scala

```diff
@@ -72,6 +74,19 @@ object ECDSASignature {
     else s
   }
 
+  def sign(message: Array[Byte], keyPair: AsymmetricCipherKeyPair): ECDSASignature = {
```

Comment:
Just a suggestion, maybe we can move that function to `crypto` package obj? This seems to be good place for this kind of functions